torch/csrc/jit/codegen/cuda/python_frontend/examples/python_example_fp16.py

```diff
@@ -0,0 +1,33 @@
+import torch
+
+from torch._C._nvfuser import FusionDefinition,Fusion,DataType
+from torch._C._nvfuser.state_creation import make_tensor,make_constant
+from torch._C._nvfuser.ops import *
+
+### Construct Fusion 
+fusion = Fusion()
+
+with FusionDefinition(fusion) as fd :
+    t0 = make_tensor(2, DataType.Half)
```

Comment:
We might want to add a tensor definition helper in `FusionDefinition` at a later point. i.e. making life easier for people to register things like `conitguous`/`broadcasting`/`size-0` e.t.c.

Wondering how we can do that conveniently while still communicating intended guarding logic. :thinking:

Reply:
Why are you suggesting in the `FusionDefinition` specifically instead of more complex helper functions?

Reply:
Just trying to have things in a single place so it's easier for users to use?

If we do expect user to be accessing `FusionDefinition()`, we might as well ask them to access everything through it? No objection if you think there's a better place to keep it.

BTW, I have similar feeling regarding our operation exposed via `torch._C._nvfuser.ops`. But I'm really just nitpicking at this point :)

Reply:
I like your idea of putting the tensor/constant definitions under `FusionDefinition`.  

I personally prefer putting the `ops` under a submodule so it is easier to query what ops exist.  However, similarly, that submodule could go under `FusionDefinition`.